tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html

```diff
@@ -686,6 +694,8 @@
         // apply.
         this._svg.selectAll("*")
             .classed("tf-hparams-parallel-coords-plot", true);
+
+        this.redrawCount += 1
```

Comment:
nit: Semicolon at end of statement, please.

(Perhaps also slightly more idiomatic as `this.redrawCount++;`, but
I don’t mind either way.)

Reply:
Thanks. Done.